### PR TITLE
Advertise change to default behavior of IModelDb.close()

### PR DESCRIPTION
**Pull request by khanaffan**
This in response to changes in PR [138877](https://dev.azure.com/bentleycs/iModelTechnologies/_git/imodel02/pullrequest/138877)

### CODE REVIEW

File: docs/changehistory/NextVersion.md [hunk 1]
```diff
@@ -29,6 +29,12 @@ This feature is currently disabled by default. Enabling it requires the use of A
    IModelApp.startup({ tileAdmin });
 ```
 
+## Change of default behavior of `IModelDb.close()` and `IModelDb.saveChanges()`
+
+By default `IModelDb.close()` use to save changes. But now it does not. Application must call `IModelDb.saveChanges()` explicitly before closing to ensure any changes made so far are committed to disk. `IModelDb.saveChanges()` also now expect to throw an exception if it fails and rollback any changes. Application code must treat exceptions as fatal an should exit afterwords. Application should not expect this to be common case and `IModelDb.saveChanges()` only fail in rare cases where there were not enough system resources to complete the operation.
```
Comment (pmconne) [inline docs/changehistory/NextVersion.md:34]: Can you clarify what happens if user closes the iModel with uncommitted changes?

**Review by pmconne — CHANGES_REQUESTED** [review]
I think you should change the target branch to `master`. Your native changes have already been integrated into addon 2.12.1.

File: docs/changehistory/NextVersion.md (hunk 1 above)
Comment (pmconne) [reply docs/changehistory/NextVersion.md]: Thanks. Do you mind if I edit this for spelling and clarity?

Comment (khanaffan) [reply docs/changehistory/NextVersion.md]: sure.

**Review by pmconne — APPROVED** [review]

**Pull request merged**